region/lowernorfair/west.json

```diff
@@ -364,28 +452,62 @@
           "to": [
             {
               "id": 2,
-              "requires": null,
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"cost": [
+                  {"heatFrames": 50}
+                ]}
+              ],
               "unlock": null
             },
             {
               "id": 4,
               "requires": [
                 {"or": [
-                  "ScrewAttack",
-                  "canUsePowerBombs",
+                  {"and": [
+                    "ScrewAttack",
+                    {"cost": [
+                      {"heatFrames": 150}
+                    ]}
+                  ]},
+                  {"and": [
+                    "canUsePowerBombs",
+                    {"cost": [
+                      {"heatFrames": 350},
+                      {"ammo": [
+                        {
+                          "type": "PowerBomb",
+                          "count": 1
+                        }
+                      ]}
+                    ]}
+                  ]},
                   {"and": [
                     "DefeatGoldenTorizo",
                     {"or": [
-                      "canIBJ",
-                      {"canShineCharge": {
-                        "usedTiles": 28
-                      }}
+                      {"and": [
+                        "canIBJ",
+                        {"cost": [
+                          {"heatFrames": 3000}
+                        ]}
+                      ]},
+                      {"and": [
+                        {"canShineCharge": {
+                          "usedTiles": 28
+                        }},
+                        {"cost": [
+                          {"heatFrames": 950},
+                          {"shinesparkFrames": 11}
+                        ]}
+                      ]}
                     ]}
                   ]}
                 ]}
               ],
-              "unlock": null,
-              "note": "The Shinespark strat requires two separate sparks"
```

Comment:
Was there something wrong with the note?

Reply:
Sometimes I don't know if the note is for me or for the end-user.

Reply:
In general I put notes as comments to explain things that I think aren't obvious. They're meant for anyone who could be reading the file.